### PR TITLE
Add commands to enable and disable Precise devmode

### PR DESCRIPTION
**Pull request by MatthewScholefield**
This adds the following two new voice commands:
 - "Enable precise development mode"
 - "Disable precise development mode"

Enabling it will start the download of the dev executable and model by changing the configuration values.

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -57,21 +64,68 @@ def handle_query_name(self, message):
...
+    @intent_file_handler("DisablePreciseDev.intent")
+    @on_error_speak_dialog('must.update')
+    def handle_use_precise_dev(self, message):
+        from mycroft.configuration.config import (
+            LocalConf, USER_CONFIG
+        )
+
+        for item in glob(expanduser('~/.mycroft/precise/precise-engine*')):
+            self.log.info('Removing: {}...'.format(item))
+            rmtree(item)
```
Comment (forslund) [inline __init__.py]: I had an issue here the glob includes a precise-engine-0.2.0.tar.gz file in the same directory as the precise-engine folder and fails asking me to update my software. (rmtree can't delete files)

Is this error correct? Is the intention that the tar.gz is included? or should something like:
```python
            if isdir(item):
                rmtree(item)
            else:
                os.remove(item)
```

be used?

Comment (MatthewScholefield) [reply __init__.py]: Good catch! Yeah, deleting all the files/folders is the intention. Fixed.

**Comment by forslund** [comment]
Another issure I found, the Disable handler uses the enable dialog (and the disable dialog seems to be missing)

**Comment by MatthewScholefield** [comment]
Should be fixed.

**Comment by forslund** [comment]
Found another tiny thing (duplicated method name), I fixed it and will be merging now.

**Pull request merged**